infor-design/enterprise-wc
# 1765 - IdsInput/IdsTextarea prevent multiple input events

**Explain the details for making this change. What existing problem does the pull request solve?**
This PR suppresses the native InputEvent so that only the webcomponent's CustomEvent fires.

**Related github/jira issue (required)**:
Closes #1765 

**Steps necessary to review your pull request (required)**:
1. Checkout branch
2. Go to http://localhost:4300/ids-textarea/example.html & http://localhost:4300/ids-input/example.html
3. In console, add an event listener for `input` to any of the input/textarea examples 
4. `$('ids-input').addEventListener('input', (evt) => console.log('Input Handler', evt));`
5. `$('ids-textarea').addEventListener('input', (evt) => console.log('Input Handler', evt));`
6. Check that the `input` event only fires once per character

***Test in Angular Forms***
1. Copy over this branch's build to the angular examples project via `npm run publish:link` or copy/pasting `dist`
2. Go to http://localhost:4200/ids-forms/example & http://localhost:4200/ids-reactive-forms/example
3. Check that these form examples still work when updating and submitting forms

***Test individual event binding in Angular***
1. Bind to an ids-input's `input` event
2. Inside [IdsInput's Component Example Class](https://github.com/infor-design/enterprise-wc-examples/blob/main/angular-ids-wc/src/app/components/ids-input/demos/example/example.component.ts), add the `log()` function below
![image](https://github.com/infor-design/enterprise-wc/assets/98347152/d6fa09c7-cab1-4bed-a744-a50b03df0ed0)
3. Inside [IdsInput's Component Example HTML](https://github.com/infor-design/enterprise-wc-examples/blob/main/angular-ids-wc/src/app/components/ids-input/demos/example/example.component.html), add input event binding
![image](https://github.com/infor-design/enterprise-wc/assets/98347152/9b4ea95e-f130-4f2d-92c2-0e4c77f5271a)
4. Type a value and check that the `input` event only logs once per character in console
5. Do the same for the `ids-textarea` angular example


**Included in this Pull Request**:
